onchain/rollups/test/foundry/util/LibServerManager.sol

```diff
@@ -0,0 +1,184 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+pragma solidity ^0.8.8;
+
+import {Vm} from "forge-std/Vm.sol";
+
+library LibServerManager {
+    using LibServerManager for string;
+    using LibServerManager for bytes32;
+    using LibServerManager for RawHash;
+    using LibServerManager for RawHash[];
+    using LibServerManager for RawOutputValidityProof;
+    using LibServerManager for RawProof;
+    using LibServerManager for RawProof[];
+
+    error InvalidOutputEnum(string);
+
+    struct RawHash {
+        bytes32 data;
+    }
```

Comment:
I'm defining the interface of `finish_epoch_response.json` so that I can decode the output of [`parseJson`](https://book.getfoundry.sh/cheatcodes/parse-json).